doc/libghdl/ASTTraversing.rst

```diff
@@ -0,0 +1,12 @@
+.. _LIB:ASTTraversing:
+
+AST Traversing
+##############
+
+If someone wants to use `libghdl`, then he probably wants to know how to traverse the AST to get the information about nodes of design units.
+It would be really nice to provide an example showing how to traverse the tree.
+In the `vhdl_ls` Load_File function is used to parse the file and get the tree root.
+The return value is an integer.
+
+* What does this integer represent?
```

Comment:
It is not an integer, but a reference to a node (which is an index so indeed an integer).  In Ada the fact that this is an integer is almost hidden, but that is clear in the low-level binding.  (Hence a reason why we should have a high-level and more pythonic API).

The value, if not null, represents an Iir_Kind_Design_File.